modules/ROOT/pages/Certificates.adoc

```diff
@@ -1,58 +1,54 @@
 = Certificate Settings
 
-All certificates in Anypoint Partner Manager are securely stored in Anypoint Platform's secrets manager, which uses a secure vault technology to store and control access to private keys, passwords, certificates, and other secrets.
+All certificates in Anypoint Partner Manager are securely stored in Anypoint Platform’s secrets manager, which uses secure vault technology to store and to control access to private keys, passwords, certificates, and other secrets.
 
-Configure your SSL certificate settings based on your certificate needs.
+You can configure your SSL certificate settings based on your certificate needs.
```

Comment:
Per style ref., try to avoid modal verbs like "can," may," and "might" unless you are trying to distinguish between a step and an option.

Reply:
Does one configure "settings"? Aren't settings themselves configuration?
And "configure your certificate based on your certificate needs" sounds kind of...unnecessary. What are we really trying to advise here?

Reply:
Fixed - superfluous statement that was consolidated into the next line.